build.gradle.kts

```diff
@@ -85,13 +85,12 @@ idea { module { generatedSourceDirs.add(jsitterMonkeyPatchSourceDir.get().asFile
  * `java.library.path`.
  *
  * This patches its source code so that we can control exactly where the tree-sitter library
- * resides. It also patches the `Tree` class so its objects can be accessed from multiple threads.
- * As we never modify trees, they are safe to be accessed.
+ * resides.
  */
 val monkeyPatchTreeSitter by
   tasks.registering(Copy::class) {
     from(zipTree(jtreeSitterSources.singleFile)) {
-      include("**/TreeSitter.java", "**/Tree.java")
+      include("**/TreeSitter.java")
```

Comment:
Quite minor, but the `filter` below can now be a simple `if` instead of a `when` (commenting here, because GH-PR-yippee)